### PR TITLE
Usability improvements for darktable-cli

### PR DESCRIPTION
**Pull request by ckreibich**
Two small tweaks to `darktable-cli`:
- Make it exit with error when a problem occurs during image conversion. You guys seem to prefer `exit()` over `return` from main, so I'm using this here too.
- Print a warning when hitting an unknown command line option. (Probably better to migrate this to `getopt` or `g_option_*` in the future...)

### CODE REVIEW

File: src/cli/main.c [hunk 1]
```diff
@@ -391,6 +391,10 @@ int main(int argc, char *arg[])
         k++;
         break;
       }
+      else
+      {
+        fprintf(stderr, _("notice: unknown option '%s'\n"), arg[k]);
```
Comment (TurboGit) [inline src/cli/main.c:396]: notice: -> warning:

Comment (ckreibich) [reply src/cli/main.c]: Done

**Review by TurboGit — APPROVED** [review]
Thanks !

**Comment by ckreibich** [comment]
Thanks for the quick turn-around!

**Pull request merged**